tests/test_add_losses.py

```diff
@@ -48,3 +51,29 @@ def test_add_losses_with_input_none():
     spectrum_in = None
     spectrum = add_losses(spectrum_in)
     assert spectrum is None
+
+
+def test_add_losses_with_peakmz_larger_precursormz():
```

Comment:
Can you please add a single sentence to tests explaining what you test here?

Reply:
done